docs/index.md

```diff
@@ -45,17 +45,11 @@ Marquez enables highly flexible [data lineage](https://en.wikipedia.org/wiki/Dat
 Marquez is a modular system and has been designed as a highly scalable, highly extensible platform-agnostic solution for metadata management. It consists of the following system components:
 
 * **Metadata Repository**: Stores all job and dataset metadata, including a complete history of job runs and job-level statistics (i.e. total runs, average runtimes, success/failures, etc).
-* **Metadata API**: RESTful API enabling a diverse set of clients to begin collecting metadata around dataset production and consumption.
+* **Metadata API**: RESTful API enabling a diverse set of clients to begin interacting with metadata around dataset production and consumption.
 * **Metadata UI**: Used for dataset discovery, connecting multiple datasets and exploring their dependency graph.
 
 <br/>
 
-<figure align="center">
-  <img src="./assets/images/design.png">
-</figure>
-
-To ease adoption and enable a diverse set of data processing applications to build metadata collection as a core requirement into their design, Marquez provides language-specific clients that implement the [Metadata API](./openapi.html). As part of our initial release, we have provided support for [Java](https://github.com/MarquezProject/marquez-java) and [Python](https://github.com/MarquezProject/marquez-python).
```

Comment:
Rather than deleting this paragraph, I would reword it to state that we provide python/java clients for _reading_ metadata.